code/game/machinery/camera/presets.dm

```diff
@@ -78,6 +82,10 @@
 
 // If you are upgrading Motion, and it isn't in the camera's New(), add it to the machines list.
 /obj/machinery/camera/proc/upgradeMotion()
+	if(isMotion())
+		return
+	if(name == initial(name))
+		name = "motion-sensitive security camera"
```

Comment:
Undocumented change.
Bit of an odd one as well to me since you can label the camera now to avoid this to happen.
Maybe instead add some text to the examine code?